core-bundle/src/EventListener/PrettyErrorScreenListener.php

```diff
@@ -107,7 +107,9 @@ private function handleException(ExceptionEvent $event): void
                 break;
 
             case $exception instanceof ServiceUnavailableHttpException:
-                $this->renderTemplate('service_unavailable', 503, $event);
+                if (!$this->renderErrorScreenByType(503, $event)) {
+                    $this->renderTemplate('service_unavailable', 503, $event);
```

Comment:
We should add a deprecation here, so we can change this to `$this->renderErrorScreenByType(503, $event);` in Contao 5. WDYT?

Reply:
I don't understand what you mean? We do want to show the template if there is no page, or you mean we should just show the generic Symfony 503 in that case?

Reply:
Exactly. Just like we do for the 4xx pages, too. You either have a page in the site structure or the generic Symfony 503.

Reply:
Hmm, I don't think thats best. We'll need the maintenance template for #3737 anyway, and as long as I do not have multiple root pages I could just reuse that template in both cases.

Reply:
I would also leave that as is :)